docs/cep/query-guide/index.md

```diff
@@ -58,8 +25,12 @@ High level syntax of StreamApp is as follows.
         ;
 ```
 
+### Syntax Example
+
 For example, this stream application with the name `Temperature-Analytics` creates a stream named `TempStream` and a query named `5minAvgQuery`.
 
+Stream workers are named by adding `@app:name('<name>')` annotation on the top of the stream worker spec. When the annotation is not added, Macrometa assigns a random UUID as the name of the stream worker.
+
```

Comment:
@diana-macrometa - The `OutputStream` is internal stream(sink) since does not have an explicit definition.
Let's replace it by `gdn` stream by adding the line below right after the other definition.
```CREATE SINK STREAM OutputStream (roomNo int, avgTemp double);```

Reply:
Done. Is this in the right place?

Btw, you can insert suggestions where you want to by clicking the icon with the plus and minus in the top of the comment editor.

Reply:
Yes, that's the right place.
Somehow it didn't give to use `+/-` previous time.